R/process_data_set_natalia.R

```diff
@@ -0,0 +1,27 @@
+library(tidyverse)
```

Comment:
Could you move this file to `data-raw/`? That's where all the data "taming" code should be.

Reply:
Note that any files included in the `R/` folder get run when building the package, in particular to convert the `roxygen2` code into the R help files in the `man/` folder.

However, the code in `process_data_set_natalia.R` is run only once to create the R package `.rda` data files in the `data/` folder. In fact, if you look at the `.Rbuildignore` file, the `data-raw/` is there telling R to ignore this folder when building the package.